test/gateway/core.js

```diff
@@ -70,12 +109,12 @@ contract('Core', function (accounts) {
         });
 
         it('should be able to commit state root and getStateRoot for given block height', async () => {
-            let response = await core.commitStateRoot(blockHeight, stateRoot, {from: worker})
-            ;
+            let response = await core.commitStateRoot(blockHeight, stateRoot, { from: worker })
```

Comment:
we should also check for the return type using `core.commitStateRoot.call`

Reply:
added